Provider io.fabric8.kubernetes.client.okhttp.OkHttpClientFactory not found

Using dekorate-bom 3.6.1 and runing mvn clean package -Ddekorate.deploy=true raise exception:
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project springboot-dekorate-app: Fatal error compiling: java.util.ServiceConfigurationError: io.fabric8.kubernetes.client.http.HttpClient$Factory: Provider io.fabric8.kubernetes.client.okhttp.OkHttpClientFactory not found -> [Help 1]

--- a/annotations/certmanager-annotations/pom.xml
+++ b/annotations/certmanager-annotations/pom.xml
@@ -169,7 +169,8 @@ limitations under the License.
                     <include>io/dekorate/certmanager/generator/**</include>
                     <include>io/dekorate/certmanager/manifest/**</include>
                     <include>io/dekorate/certmanager/model/**</include>
-                    <include>META-INF/services/*</include>
+                    <include>META-INF/services/io.dekorate.*</include>
+                    <include>META-INF/services/javax.annotation.processing.Processor</include>
                     <include>META-INF/MANIFEST.MF</include>
                   </includes>
                 </filter>

--- a/annotations/kind-annotations/pom.xml
+++ b/annotations/kind-annotations/pom.xml
@@ -172,7 +172,8 @@ limitations under the License.
                     <include>io/dekorate/kind/manifest/**</include>
                     <include>io/dekorate/kind/hook/**</include>
                     <include>io/dekorate/kind/listener/**</include>
-                    <include>META-INF/services/*</include>
+                    <include>META-INF/services/io.dekorate.*</include>
+                    <include>META-INF/services/javax.annotation.processing.Processor</include>
                     <include>META-INF/MANIFEST.MF</include>
                   </includes>
                 </filter>

--- a/annotations/knative-annotations/pom.xml
+++ b/annotations/knative-annotations/pom.xml
@@ -166,7 +166,8 @@ limitations under the License.
                     <include>io/dekorate/knative/listener/**</include>
                     <include>io/dekorate/knative/manifest/**</include>
                     <include>io/dekorate/knative/util/**</include>
-                    <include>META-INF/services/*</include>
+                    <include>META-INF/services/io.dekorate.*</include>
+                    <include>META-INF/services/javax.annotation.processing.Processor</include>
                     <include>META-INF/MANIFEST.MF</include>
                   </includes>
                 </filter>

--- a/annotations/kubernetes-annotations/pom.xml
+++ b/annotations/kubernetes-annotations/pom.xml
@@ -167,7 +167,8 @@ limitations under the License.
                     <include>io/dekorate/kubernetes/manifest/**</include>
                     <include>io/dekorate/kubernetes/hook/**</include>
                     <include>io/dekorate/kubernetes/listener/**</include>
-                    <include>META-INF/services/*</include>
+                    <include>META-INF/services/io.dekorate.*</include>
+                    <include>META-INF/services/javax.annotation.processing.Processor</include>
                     <include>META-INF/MANIFEST.MF</include>
                   </includes>
                 </filter>

--- a/annotations/minikube-annotations/pom.xml
+++ b/annotations/minikube-annotations/pom.xml
@@ -170,7 +170,8 @@ limitations under the License.
                     <include>io/dekorate/minikube/configurator/**</include>
                     <include>io/dekorate/minikube/decorator/**</include>
                     <include>io/dekorate/minikube/manifest/**</include>
-                    <include>META-INF/services/*</include>
+                    <include>META-INF/services/io.dekorate.*</include>
+                    <include>META-INF/services/javax.annotation.processing.Processor</include>
                     <include>META-INF/MANIFEST.MF</include>
                   </includes>
                 </filter>

--- a/annotations/openshift-annotations/pom.xml
+++ b/annotations/openshift-annotations/pom.xml
@@ -166,7 +166,8 @@ limitations under the License.
                     <include>io/dekorate/openshift/listener/**</include>
                     <include>io/dekorate/openshift/util/**</include>
                     <include>io/dekorate/openshift/*</include>
-                    <include>META-INF/services/*</include>
+                    <include>META-INF/services/io.dekorate.*</include>
+                    <include>META-INF/services/javax.annotation.processing.Processor</include>
                     <include>META-INF/MANIFEST.MF</include>
                   </includes>
                 </filter>

--- a/annotations/prometheus-annotations/pom.xml
+++ b/annotations/prometheus-annotations/pom.xml
@@ -131,7 +131,8 @@
                     <include>io/dekorate/prometheus/decorator/**</include>
                     <include>io/dekorate/prometheus/manifest/**</include>
                     <include>io/dekorate/prometheus/model/**</include>
-                    <include>META-INF/services/*</include>
+                    <include>META-INF/services/io.dekorate.*</include>
+                    <include>META-INF/services/javax.annotation.processing.Processor</include>
                     <include>META-INF/MANIFEST.MF</include>
                   </includes>
                 </filter>

--- a/annotations/servicebinding-annotations/pom.xml
+++ b/annotations/servicebinding-annotations/pom.xml
@@ -177,7 +177,8 @@ limitations under the License.
                     <include>io/dekorate/servicebinding/generator/**</include>
                     <include>io/dekorate/servicebinding/manifest/**</include>
                     <include>io/dekorate/servicebinding/model/**</include>
-                    <include>META-INF/services/*</include>
+                    <include>META-INF/services/io.dekorate.*</include>
+                    <include>META-INF/services/javax.annotation.processing.Processor</include>
                     <include>META-INF/MANIFEST.MF</include>
                   </includes>
                 </filter>

--- a/core/pom.xml
+++ b/core/pom.xml
@@ -195,7 +195,8 @@ limitations under the License.
                     <include>io/dekorate/processor/**</include>
                     <include>io/dekorate/project/**</include>
                     <include>io/dekorate/utils/**</include>
-                    <include>META-INF/services/*</include>
+                    <include>META-INF/services/io.dekorate.*</include>
+                    <include>META-INF/services/javax.annotation.processing.Processor</include>
                     <include>META-INF/MANIFEST.MF</include>
                   </includes>
                 </filter>

